paasta_tools/kubernetes_tools.py

```diff
@@ -2707,6 +2711,39 @@ def ensure_paasta_api_rolebinding(kube_client: KubeClient, namespace: str) -> No
         )
 
 
+def ensure_paasta_namespace_limits(kube_client: KubeClient, namespace: str) -> None:
+    limits = get_all_limit_ranges(kube_client, namespace=namespace)
+    limits_names = [item.metadata.name for item in limits]
```

Comment:
super-nit: this probably doesn't matter since there's probably only ever going to be a single limit range, but I personally like to make sure that anything i use `in` on is a set just in case :)

```suggestion
    limits_names = {item.metadata.name for item in limits}
```